src/platform/user/selectors.js

```diff
@@ -35,3 +37,13 @@ export const selectVet360ResidentialAddress = state =>
 export function createIsServiceAvailableSelector(service) {
   return state => selectAvailableServices(state).includes(service);
 }
+
+export const selectFilteredPatientFacilities = state =>
+  selectPatientFacilities(state).filter(f => !f.facilityId.startsWith('742')) ||
```

Comment:
Should this be `selectPatientFacilities(state)?.filter`? Looks like we're doing that elsewhere.